lib/Controllers/list.js

```diff
@@ -68,6 +69,14 @@ List.prototype.fetch = function(req, res, context) {
         order.push([sortColumn, 'ASC']);
       }
     });
+    var allowedColumns = this.resource.sort.attributes || Object.keys(model.rawAttributes);
+    var columnNames = order.map(_.first);
+    var disallowedColumns = _.difference(columnNames, allowedColumns);
+    if(disallowedColumns.length) {
+      res.status(400);
+      res.json({error: 'Sorting is not allowed on the attributes: ' + disallowedColumns.join(', ') });
```

Comment:
I'm not one to say, I've only just looked at this this afternoon :P Looking at the delete.js as an example just now though I also noticed it used the following:

```javascript
res.status(500);
context.error = {
  message: 'internal error',
  errors: [ err.message ]
};
return context.continue();
```
as it's way of showing an error so I changed the 400 to use the same style